kentik_api_library/kentik_api/kentik_api.py

```diff
@@ -42,7 +43,7 @@ def __init__(
             logging.debug("KentikAPI: null api_host, setting to %s", self.API_HOST_US)
             api_host = self.API_HOST_US
         api_v5_url = self.make_api_v5_url(api_host)
-        connector = APIConnector(api_v5_url, auth_email, auth_token, timeout, retry_strategy, proxy)
+        connector = APIConnector(api_v5_url, auth_email, auth_token, client_version, timeout, retry_strategy, proxy)
```

Comment:
Let's retrieve the version in `APIConnector` as opposed to passing it as an argument to the constructor.

Reply:
Updating